docs/index.rst

```diff
@@ -18,7 +18,7 @@
 
 
 
-.. image:: img/pin_large.png
+.. image:: ../airflow/www/static/pin_large.png
```

Comment:
.... I think this should fail to build the docs - I've removed that file. Why isn't it?

Reply:
Oh no, I just made it a slightly smaller file.